### PR TITLE
Ecosystem contributions

### PR DESCRIPTION
**Pull request by Nyco**
Our contributions to the ecosystem, including our pieces, and third-party opensource projects

### CODE REVIEW

**Comment by michalwski** [comment]
Our XML parser [esl/exml](https://github.com/esl/exml) is missing.

**Comment by ramabit** [comment]
The Smack part is ok

File: doc/Contributions.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+* XEP-0313: Message Archive Management: https://github.com/robbiehanson/XMPPFramework/pull/733
+* XEP-0030: Service Discovery: https://github.com/robbiehanson/XMPPFramework/pull/736
+* MUC light: https://github.com/robbiehanson/XMPPFramework/pull/750
+* Token-based reconnection: https://github.com/robbiehanson/XMPPFramework/pull/758
+* Revamped README: making people feel like this is a well mantained and up to date framework
+* Created a way to Mock a piece of the framework to improve the way we write tests
+
+### Smack for Android
+
+https://github.com/igniterealtime/Smack
```
Comment (michalwski) [inline doc/Contributions.md]: Probably it the following would be enough:
`[igniterealtime/Smack](https://github.com/igniterealtime/Smack)`

Comment (michalwski) [inline doc/Contributions.md]: The same applies to other links to external products.

File: doc/Contributions.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+https://github.com/movim/movim
+
+* Docker image for Movim: https://github.com/esl/movim-docker
+
+## Developped and maintained by Erlang Solutions
+
+### escalus
+
+Available on: [github.com/esl/escalus](https://github.com/esl/escalus)
```
Comment (michalwski) [inline doc/Contributions.md]: If we want to present the whole URL then let's be consistent and put `http(s)://` prefix everywhere.

File: doc/README.md [hunk 3]
```diff
@@ -7,6 +7,7 @@
         * Build MongooseIM on a supported OS
         * Perform basic configuration
         * Use the main administration script, `mongooseimctl`
+* Contributions
```
Comment (michalwski) [inline doc/README.md]: Should this be a link to the Contributins.md file?

**Pull request merged**